service/src/main/kotlin/fi/espoo/evaka/daycare/PreschoolTerms.kt

```diff
@@ -58,6 +59,23 @@ fun Database.Read.getPreschoolTerms(): List<PreschoolTerm> {
         .toList<PreschoolTerm>()
 }
 
+fun Database.Read.getPreschoolTerm(id: PreschoolTermId): PreschoolTerm? =
+    this.createQuery(
+            """
+        SELECT
+            id,
+            finnish_preschool,
+            swedish_preschool,
+            extended_term,
+            application_period,
+            term_breaks
+        FROM preschool_term 
+        WHERE id = :id
+        """
+        )
+        .bind("id", id)
```

Comment:
would prefer the new bind syntax in queries

Reply:
I fixed also rest of the preschool term queries to have this syntax.